recipes/llnl-units/all/conanfile.py

```diff
@@ -0,0 +1,106 @@
+import os
+
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeToolchain
+from conan.tools.files import copy, rm, rmdir, get
+
+
+class UnitsConan(ConanFile):
+    name = "llnl-units"
+    license = "BSD-3"
```

Comment:
```suggestion
    license = "BSD-3-Clause"
```

Please, always use the short for listed by SPDX: https://spdx.org/licenses/BSD-3-Clause.html